### ISSUE TITLE
Question regarding the use of nested object. 

### ISSUE THREAD

**Issue by melvinalmonte** [massmutual/react-collapsing-table#79]
I'm new to react so far I've loved using this library. I have a question. How do you handle accessing data on a nested object? for example, I've got the following object. 

```javascript
let data = [{
      "id": 4,
      "Rows": [{ "name": "D", "points": 20, "percent": 80 }]
    }]
 ```

I'd like to access the values in Row. How do you go around achieving this?

I'm thinking about the following approach to set up my columns but it's not working. 

```javascript

const columns = [ {
      accessor: "id",
      label: "ID",
      priorityLevel: 1,
      position: 1,
      minWidth: 150
    },
    {
      accessor: "Rows.name",
      label: "Name",
      priorityLevel: 2,
      position: 2,
      minWidth: 150
    }])
```
All I get is an empty row. How could I go about this? any help would be greatly appreciated.

**Comment by Salinn** [comment]
Hi @melvinalmonte is there a reason you nested your rows like that? Typically the row is constructed as a non nested object. 

One alternative that you could do is take your rows and flatten them
``` 
const rows = [{
  "id": 4,
  "Rows": [{ "name": "D", "points": 20, "percent": 80 }]
}]

const newRows = rows.map(row => ({ ...row, ...row["Rows"][0] }))

//Now the new rows would look like this
const data = [{ 
  "id": 4,
  "Rows": [{ "name": "D", "points": 20, "percent": 80 }]
  "name": "D", 
  "points": 20, 
  "percent": 80
}]
```

**Comment by melvinalmonte** [comment]
Yeah, unfortunately, I'm working with a call that returns the data in this format. I'll give your suggestion a try :)

**Comment by Salinn** [comment]
@melvinalmonte will the "Rows" attribute on your object ever return more than one item in the array or is there more data on the object? 

If it will return more than one row then that could be an issue

**Comment by melvinalmonte** [comment]
Yes, it will return more than one row.

**Comment by Salinn** [comment]
So it would be more like this?

const rows = [
  {
    "id": 4,
    "Rows": [
      { "name": "A", "points": 100, "percent": 0 },
      { "name": "B", "points": 80, "percent": 20 },
      { "name": "C", "points": 60, "percent": 40 },
      { "name": "D", "points": 20, "percent": 80 },
    ]
  },
  {
    "id": 5,
    "Rows": [
      { "name": "A", "points": 70, "percent": 30 },
      { "name": "B", "points": 50, "percent": 50 },
      { "name": "C", "points": 30, "percent": 70 },
      { "name": "D", "points": 10, "percent": 90 },
    ]
  }
]

**Comment by melvinalmonte** [comment]
The first one

**Comment by Salinn** [comment]
I can 100% say that is sadly something this library is not built for =(. Also I don't recommend my earlier map suggestion as that won't work if you have more than one row. 

Out of curiosity what are you trying to display that has nested rows like that where they all need to be visible at once? If you could have them be separate tables you could do something like this. 
```
const columns = [
  {
      accessor: "id",
      label: "ID",
      priorityLevel: 1,
      position: 1,
      minWidth: 150
  },
  {
      accessor: "name",
      label: "Name",
      priorityLevel: 2,
      position: 2,
      minWidth: 150
  },
  {
      accessor: "points",
      label: "Points",
      priorityLevel: 3,
      position: 3,
      minWidth: 150
  }
])
const rows = [
  {
    "id": 4,
    "Rows": [
      { "name": "A", "points": 100, "percent": 0 },
      { "name": "B", "points": 80, "percent": 20 },
      { "name": "C", "points": 60, "percent": 40 },
      { "name": "D", "points": 20, "percent": 80 },
    ]
  },
  {
    "id": 5,
    "Rows": [
      { "name": "A", "points": 70, "percent": 30 },
      { "name": "B", "points": 50, "percent": 50 },
      { "name": "C", "points": 30, "percent": 70 },
      { "name": "D", "points": 10, "percent": 90 },
    ]
  }
]

const tables = rows.map(row => {
  return (
    <ReactCollapsingTable rows={ row.Rows } columns={ columns } />
  )
})
```

This would create multiple tables with all of the data displayed for each

**Comment by melvinalmonte** [comment]
It's for an internal tool at my job which returns an object with a lot of consolidated data around a user. The table would have to display information found in other objects that look just like this one. I ended up building my own collapsible table to take care of how uniquely structured the data I'm receiving is. I wanted to give this a crack given how great this worked on another project I worked on. I don't think the multiple table approach would work, again, given how the data is organized. Thank you for your help. I really appreciate it. Keep up the good work :)
